src/views/OtherStoresInventoryModal.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal">
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Other stores inventory") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content class="ion-padding">
+    <ion-list v-if="otherStoresInventoryDetails.length">
```

Comment:
In case if there are no available inventory for other stores then I think we can display an empty state.

Reply:
Updated sir